### DISCUSSION TITLE
How to show/hide regions without reloading?

### DISCUSSION THREAD

**Discussion by egorksv** [katspaugh/wavesurfer.js#3752] [Q&A]
Hi,

We have a massive track (think 6-hours stream) with 400-500 AI-generated regions. When it comes to manual editing and adjusting in post-processing, there is a need to quickly show/hide all regions - is there a way to do that without clearing and re-creating all regions?

I've seen some thing about "opaque regions" in other threads but couldn't find any specifics on that.

Many thanks!

**Comment by katspaugh** [comment]
`addRegion` returns a region object which you can later remove with `region.remove()`.
